Add Grafana Guide, fixes #33

tuminzee opened the pull request:
Fixes https://github.com/FrigadeHQ/trench/issues/33

Hi @SSmale and @christianmat,

I've made some changes to simplify the steps and omitted certain parts to keep the guide clean. Please review and let me know if you’d like any adjustments to the tone or the direction of the guide.

adding ss for you ref


<img width="1624" alt="SCR-20241220-dsji" src="https://github.com/user-attachments/assets/d8262b4c-def5-47ed-8c7b-d95c6502b04f" />
<img width="1624" alt="SCR-20241220-dsjy" src="https://github.com/user-attachments/assets/83bbd37e-e792-41eb-8fa2-2f3b5173d75f" />
<img width="1624" alt="SCR-20241220-dskj" src="https://github.com/user-attachments/assets/bdfba286-72f7-4f00-b0d9-55be5898fe4e" />
<img width="1624" alt="SCR-20241220-dskv" src="https://github.com/user-attachments/assets/96337a2a-db13-4b87-b95c-76fd82a72b4a" />
<img width="1624" alt="SCR-20241220-dsls" src="https://github.com/user-attachments/assets/e9dd4bb4-0156-42c1-bf20-964a6e802619" />
<img width="1624" alt="SCR-20241220-dsmb" src="https://github.com/user-attachments/assets/5eaa4093-6b54-4b5c-a37b-e3a1a3ff64b7" />
<img width="1624" alt="SCR-20241220-dsml" src="https://github.com/user-attachments/assets/3ba04758-8fe2-4208-9ef7-f0c0ba682010" />

tuminzee commented:
p.s. I think these are a lot of ss 😆

SSmale reviewed (commented):
Looks good to me. 

Is the dashboard code collapsed by default?

tuminzee commented:
@SSmale I'm not sure I understand your comment. Could you tell me more about what you mean by 'Is the dashboard code collapsed by default?'

tuminzee commented:
@SSmale also thank you very much for the brief issue. I used most of the comments in the guide 🕺🏼

tuminzee commented:
ah cool cool, this is great let me look how to do this in mintlify
will update the PR

SSmale commented:
For completeness and to possibly save your sanity, I do not know if it is possible in this projects doc engine.

tuminzee commented:
commit ref https://github.com/FrigadeHQ/trench/pull/41/commits/4d45a903eedfd55a3d46dda8cedd72b40f1036e3

I tried it it doesn't seem to work in mintlify.
Although I have one observation, the most annoying part right now in the guide is the dashboard json
I can link up the json file which could be downloaded directly? what do you think of that? other parts of the guide are small enough to be present in the same page

<img width="861" alt="image" src="https://github.com/user-attachments/assets/254ea776-6e43-49a8-bb75-1b1f453310ee" />


@christianmat I was not able to link json with relative path imports so for this above image demo. I added the deteminsitic link where the file would be present in the future https://raw.githubusercontent.com/FrigadeHQ/trench/refs/heads/main/apps/docs/json/grafana-dashboard.json (the file would only be available after the merge in the main branch)

let me know what do you think about this approach @SSmale

Also I changed markdown steps to https://mintlify.com/docs/content/components/steps#steps
downside of using mintlify steps is that I was not able to tag steps in url so it might because a problem to share a specific step

christianmat commented:
The deep link looks like a good solution to me. Great work. Waiting for @SSmale to confirm it looks good and then we can merge.

SSmale commented:
Yeah looks like a great solution!
